### ISSUE TITLE
ToolTips for pedestal-Items missing

### ISSUE THREAD

**Issue by MaPePeR** [sinkillerj/ProjectE#596]
A lot of the Items that now have a description for what they do in the pedestal do not have a description for what they do when not in the pedestal, so it feels like they are pedestal-only

Life/Body/Soulstone

Also they don't have a 'how to activate'-help

**Comment by Brycey92** [comment]
Speaking of this, would it be plausible to add an NEI handler to ProjectE items to provide this information, similar to what's done in Applied Energistics 2 and Extra Utilities?

**Comment by MaPePeR** [comment]
You mean: showing emc values for blocks in world? I think sinkillerj was trying something like this

**Comment by Brycey92** [comment]
No, I mean something like this http://i.imgur.com/9VZbk6m.png or this http://i.imgur.com/0wCgFuB.png

**Comment by MaPePeR** [comment]
ah, right. that would be nice

**Comment by williewillus** [comment]
I don't know how to handle it, because so many items have diff functions (in bag, chest, inventory/hand, pedestal...) we're going to have tool tip overload if we add that to everything

**Comment by Brycey92** [comment]
That's why you keep the tooltips brief and put everything else in the NEI handler. I believe the ones AE2 and EU use can have as many pages as they need per-item.

**Comment by williewillus** [comment]
Yeah an ExU kind of NEI page that states the function would be better

**Comment by sinkillerj** [comment]
Better solution I feel is still a help guide, think Lexica Botania.

**Comment by Brycey92** [comment]
Ooh, yeah a unified ingame guide could be even better!

**Comment by MaPePeR** [comment]
I still think we need those tooltips. Maybe only show them on sneak or something like this.

New Players (or bacon_donut in todays stream) get very confused about what the item can do at the moment, because they only see the tooltop for the pedestal. (Only show that in the pedestal gui or on sneak should maybe be the default)
